### ISSUE TITLE
init calls should offer callbacks, init calls should offer local settings

### ISSUE THREAD

**Issue by andre-d** [wet-boew/wet-boew#5769]
For example, I should be able to be able to init datatables with specific settings I currently have to do so with window['wb-tables'], when it could really just exist inside the init event.  If I want to init two different dynamic datatables, I must wait until the datatable is inited via a callback in the first datatable.  

The point is though, I currently have to depend on the individual callbacks of the specific libraries I am initing (if they exist at all), which is not a good idea.  All init calls should have local callbacks, all init calls should have local settings.  if you want to init two complex dynamic datatables (and still get the advantages the i18n/etc provides), you can see the problem, window['wb-tables'] is not local to the specific table.  Additionally,if you wait for the first table to be inited, you have to use a datatables specific callback.

Having local settings does not fix the need for local callbacks though.  In some situations, it would be nice to be able to do something like (below), or be aware when the specific dependency is ready, etc.
```js
$('body').on('click', '.wb-lbx:not(.wb-lbx-inited)', function (e) {
    e.preventDefault();
    $(this).trigger('wb-wb-init.wb-lbx', {callback: function() {$(this).trigger('open.wb-lbx')'}});
})
```

**Comment by hsrudnicki** [comment]
Can be this issue closed or otherwise?

**Comment by hsrudnicki** [comment]
wait for v4.1.0 or if otherwise, please update this issue's status as well

**Comment by hsrudnicki** [comment]
sorry by mistake closed, this issue needs to be opened till v4.1.0

**Comment by masterbee** [comment]
Closing this issue since this is very similar in the direction of the reworking of the WET library  - which can be followed in the wet-blew-experimental repository

/cc @LaurentGoderre
